enclave-tls/src/verifiers/sgx-ecdsa/main.c

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2020-2021 Alibaba Cloud and Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <enclave-tls/verifier.h>
+#include <enclave-tls/log.h>
+
+extern enclave_verifier_err_t enclave_verifier_register(enclave_verifier_opts_t *opts);
+extern enclave_verifier_err_t sgx_ecdsa_verifier_pre_init(void);
+extern enclave_verifier_err_t sgx_ecdsa_verifier_init(enclave_verifier_ctx_t *ctx,
+						      enclave_tls_cert_algo_t algo);
+//extern enclave_verifier_err_t null_extend_cert(enclave_verifier_ctx_t *ctx,
+//					      const enclave_tls_cert_info_t *cert_info);
```

Comment:
extend_cert is not used by any verifier instance.

Reply:
sure